CHANGELOG.rst

```diff
@@ -14,6 +14,17 @@ Change Log
 Unreleased
 ----------
 
+[4.9.0] - 2022-05-19
+--------------------
+
+Changed
+~~~~~~~
+
+* Corrupt cookie logging:
+
+  * Now independent of other cookie logging; no longer needs to meet cookie size threshold or sampling rate.
+  * New setting ``UNUSUAL_COOKIE_SAMPLING_LOG_CHUNK`` helps avoid truncated (non-decryptable) messages by splitting the output across multiple log messages.
```

Comment:
For these two settings, why do they use the term `SAMPLING`? Is that just copy/paste, or because you think there might be other unusual cookies? I think I'd like `UNUSUAL_COOKIE_HEADER_XXX` better.

Reply:
Oh, yeah, that's just left over from when it was gated on sampling! I'll rename. (And that means this will be off by default; I'll need to rename in the config repo as well.)

Reply:
I'm thinking `UNUSUAL_COOKIE_HEADER_{PUBLIC_KEY,LOG_CHUNK}`

Reply:
This line is still using the old name.